dist/profile/templates/jenkinscontroller/casc/clouds.yaml.erb

```diff
@@ -1,14 +1,15 @@
 <%- if @jcasc['cloud_agents'] && @jcasc['cloud_agents']['disabled'].to_s != "true" -%>
 jenkins:
   clouds:
-  <%- if @jcasc['cloud_agents']['azure-vm-agents'] && !@jcasc['cloud_agents']['azure-vm-agents'].empty? -%>
+  <%- if @jcasc['cloud_agents']['azure-vm-agents'] && @jcasc['cloud_agents']['azure-vm-agents']['azureVMClouds'] -%>
```

Comment:
The `empty` condition wasn't needed after all?

Reply:
It's not needed now that the next instruction is an `each` loop which handles the emptiness case.